import-export-cli/credentials/credentials.go

```diff
@@ -47,6 +47,8 @@ type Credential struct {
 type Credentials struct {
 	// Environments specific credentials
 	Environments map[string]Environment `json:"environments"`
+	// Microgateway Adapter Environments specific credentials
+	MgwAdapterEnvs map[string]MgToken `json:"mgw_adapter_envs"`
```

Comment:
Can we change `json:"mgw_adapter_envs"` to camel case to make it consistent with the way we are naming json keys elsewhere?

Reply:
```
config:
  http_request_timeout: 10000
  export_directory: /home/subhashinie/.wso2apictl/exported
  kubernetes_mode: false
  token_type: JWT
  vcs_deletion_enabled: false
  vcs_config_file_path: ""
  tls-renegotiation-mode: never
environments:
  dev:
    apim: someurl
    publisher: ""
    devportal: ""
    registration: ""
    admin: ""
    token: someurl/oauth2/token
    mi: ""
mgw_adapter_envs:
  dev:
    adapter: anotherurl
```
This is how the config looks like after adding envs.  Which is why I used this convention.